apps/client/src/components/atoms/text/Text.component.tsx

```diff
@@ -0,0 +1,144 @@
+import { ReactElement } from 'react';
+// Interfaces
+import { TextSizeTag, TextTypeTag, TextVariant } from '../../../models';
+// Constants
+import { TextStyles } from '../../../constants';
+// Styles
+import './text.css';
+
+interface TextProps {
+  type: TextTypeTag;
+  size: TextSizeTag;
+  // eslint-disable-next-line react/require-default-props
+  colorClass?: string;
+  children: string;
+}
+
+export function Text({
```

Comment:
There is too much repetition in this component.

Looking into something like this:
function TextContainer(variant: TextVariant){
    if(type === TextVariant.H1) return h1;
    if(type === TextVariant.H2) return h2;
    if(type === TextVariant.Paragraph) return p;
    ....
}

function Text({ ..., children} ){
   return <TextContainer variant={textVariant} {... other common styles}>{children}</TextContainer>
}

Reply:
refactored text component https://github.com/the-ai-team/razor/pull/51/commits/9953c3957419625c23d0ca0cccf993ac3ea6eea9